src/MultiTypeSymEntry.chpl

```diff
@@ -393,6 +425,11 @@ module MultiTypeSymEntry
         return (entry: borrowed SegStringSymEntry);
     }
 
+    proc toCompositeSymEntry(entry: borrowed AbstractSymEntry) throws {
+        // TODO - need a way to manipulate this to give the type
+        return (entry: borrowed CompositeSymEntry);
+    }
+
```

Comment:
I don't think the parens are required, just probably left over from something. 

And no because the return type of the function cannot be figured out by the compiler due to the SegArraySymEntry being a generic type.